src/app/settings/panel-settings/components/edit-panel-configuration/edit-panel-configuration.component.ts

```diff
@@ -0,0 +1,146 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { FormBuilder, FormGroup, Validators } from '@angular/forms'
+import { PanelConfiguration } from '../../models/panel-configuration'
+import { PanelModel, PanelType, SkaarhojModel } from '../../models/panel-enums'
+import { SelectOption } from '../../../../shared/models/select-option'
+import { PanelLayoutConfiguration } from '../../models/panel-layout-configuration'
+import { PanelLayoutConfigurationStateService } from '../../services/panel-layout-configuration-state.service'
+import { PanelTypeAndModelValidatorService } from '../../services/panel-type-and-model-validator.service'
+
+const PANEL_GROUP_ID: string = 'DEFAULT_PANEL_GROUP_ID'
+
+@Component({
+  selector: 'sofie-edit-panel-configuration',
+  templateUrl: './edit-panel-configuration.component.html',
+})
+export class EditPanelConfigurationComponent implements OnInit {
+  @Input() public panelConfiguration?: PanelConfiguration
+
+  @Output() public onSave: EventEmitter<PanelConfiguration> = new EventEmitter()
+  @Output() public onCancel: EventEmitter<void> = new EventEmitter()
+
+  public form: FormGroup
+
+  public typeOptions: SelectOption<PanelType>[] = [
+    {
+      name: PanelType.SKAARHOJ,
+      value: PanelType.SKAARHOJ,
+    },
+  ]
+
+  public modelOptions: SelectOption<PanelModel>[] = []
+
+  public panelLayoutConfigurationOptions: SelectOption<string>[] = []
+
+  constructor(
+    private readonly formBuilder: FormBuilder,
```

Comment:
```suggestion
    private readonly formBuilder: NonNullableFormBuilder,
```

If you use the NonNullableFormBuilder you don't have to use this.formBuilder.nonNullable below :)

Reply:
It's been updated.